mcs/class/Facades/subdirs.make

```diff
@@ -41,6 +41,9 @@ netstandard_drawing_SUBDIRS = System.Drawing.Primitives netstandard
 monotouch_SUBDIRS = $(common_DEPS_SUBDIRS) $(mobile_only_DEPS_SUBDIRS)
 monotouch_PARALLEL_SUBDIRS = $(common_SUBDIRS) $(mobile_only_SUBDIRS)
 
+monodroid_SUBDIRS = $(monotouch_SUBDIRS) netstandard
```

Comment:
we should do the Mono.Android reference changes for System.Drawing.Primitives too and then just use `monodroid_SUBDIRS = $(monotouch_SUBDIRS) $(netstandard_drawing_SUBDIRS)` here

Reply:
Sure, that works too. Though netstandard might require more types in the future (I've seen plans somewhere) that won't ever be in System.Drawing.Primitives so I'd prefer just refing Mono.Android.dll from netstandard.dll.

Reply:
Let's just use it `System.Drawing.Primitives` that one we can easily fix also XI and we are more compatible with official package